index.html

```diff
@@ -4292,6 +4306,493 @@ <h3>Proof Formats</h3>
+          </li>
+          <li>
+a verified document (object <var>document</var>)
+          </li>
+          <li>
+a media type (string <var>mediaType</var>)
+          </li>
+          <li>
+zero or more warnings (array of <a>ProblemDetails</a> <var>warnings</var>)
+          </li>
+          <li>
+zero or more errors (array of <a>ProblemDetails</a> <var>errors</var>)
+          </li>
+        </ul>
+
+        <p>
+The securing mechanism verification algorithm is as follows:
+        </p>
+
+        <ol class="algorithm">
+          <li>
+Set <var>securingMechanismVerified</var> to an empty object and initialize the
+following fields: <var>status</var> to `false`, <var>warnings</var> to an
+empty array, and <var>errors</var> to an empty array.
+          </li>
+          <li>
+Determine the specific mechanism for extracting the <a>conforming document</a>
+(<var>extractDocument</var>):
+            <ol class="algorithm">
+              <li>
+If the <var>inputMediaType</var> is `application/vc+ld+json` or
+`application/vp+ld+json`, the <var>extractDocument</var> function takes
+<var>inputBytes</var> as an input parameter, parses the value as JSON into
+an object, removes the `proof` property, and returns the result as an object.
+If JSON parsing fails, a <a href="#PARSING_ERROR">PARSING_ERROR</a> is appended
+to <var>securingMechanismVerified</var>.<var>errors</var>.
+              </li>
+              <li>
+If the <var>inputMediaType</var> is associated with the JOSE or COSE
+specifications, the [[?VC-JOSE-COSE]] specification defines a process for
+extracting the <a>conforming document</a>.
+<div class="issue" title="Document extraction process in vc-jose-cose">
+The process for extracting a conforming document is not clearly documented in
+the [[?VC-JOSE-COSE]] specification yet, but once it is, a link to that section
+of the specification will be included in this text.
+</div>
+              </li>
+              <li>
+Other extracting functions are possible, but need not be listed in
+the <a data-cite="?VC-SPECS#securing-mechanisms">Securing Mechanisms</a>
+section of the Verifiable Credentials Specifications Directory [[?VC-SPECS]].
+Implementers might find these other specifications through other standards
+bodies or communities that utilize this specification.
+              </li>
+            </ol>
+          </li>
+          <li>
+Determine the specific cryptographic verification algorithm
+(<var>verifyProof</var>) required to verify the securing mechanism:
+            <ol class="algorithm">
+              <li>
+If the <var>inputMediaType</var> is `application/vc+ld+json` or
+`application/vp+ld+json`, the Verifiable Credential Data Integrity
+Specification [[?VC-DATA-INTEGRITY]] defines the
+<a data-cite="VC-DATA-INTEGRITY#verify-proof">process for identifying
+a specific cryptography suite type</a> used for cryptographic verification by
+utilizing the `proof`.`type` property. A list of registered proof types can be
```

Comment:
Please remove the external references to vc-data-integrity.  This is a layering violation.

It's fine for a verification algorithm in VCDM to say how to verify the things defined in the VCDM.  But it's the job of the securing specifications to define how to verify the things defined there.  They shouldn't be intermingled.

Reply:
Agree, see comment above https://github.com/w3c/vc-data-model/pull/1338/files#r1411404530

Reply:
@selfissued wrote:
> Please remove the external references to vc-data-integrity. This is a layering violation.

I take it you will want all external references to vc-jose-cose to be removed as well, then?

... and if we do that, how are people supposed to understand which specification to use for what purpose?

@jyasskin if we removed all references to the securing specifications, would that cause concern for you?

I'm fine w/ pushing details to the securing specs and making this more generic. My concern is that @jyasskin was asking for more detail, not less, so some balance needs to be struck that @OR13, @selfissued, and @jyasskin can agree to.

Please endeavor to figure out what is going to work for each of you and then propose something. If I do what @OR13 and @selfissued want, I expect others to object because there won't be a clear line from this algorithm to ANY of the securing specifications. 

We can get there either through the VC Specifications Directory or we can get there through direct references to the normative securing specifications that this group is creating. We can eliminate both paths and leave it as an undocumented exercise for the implementer, which was one of @jyasskin's initial concerns (which I agree with).